src/lib.rs

```diff
@@ -673,59 +674,30 @@ impl Wizer {
 
     // NB: keep this in sync with the Wasmtime config.
     fn wasm_features(&self) -> wasmparser::WasmFeatures {
-        wasmparser::WasmFeatures {
-            mutable_global: true,
-            saturating_float_to_int: true,
-            sign_extension: true,
-            floats: true,
-            component_model_values: false,
-            component_model_nested_names: false,
-
-            // Proposals that we support.
-            multi_memory: self.wasm_multi_memory.unwrap_or(DEFAULT_WASM_MULTI_MEMORY),
-            multi_value: self.wasm_multi_value.unwrap_or(DEFAULT_WASM_MULTI_VALUE),
-
-            // Proposals that we should add support for.
-            reference_types: self
-                .wasm_reference_types
+        let mut features = WasmFeatures::WASM2;
```

Comment:
Did you verify that this matches the feature set we had before?

Reply:
See https://github.com/bytecodealliance/wasm-tools/blob/v1.228.0/crates/wasmparser/src/features.rs#L278 it is. I've also enabled tail-call and extended-const since they should be safe for wizer, but if you prefer I can add options to make it possible to disable these two

Reply:
Seems fine